test/Microsoft.AspNet.Mvc.Xml.Test/XmlSerializerOutputFormatterTest.cs

```diff
@@ -228,7 +228,7 @@ public void DefaultConstructor_ExpectedWriterSettings_Created()
             var body = outputFormatterContext.HttpContext.Response.Body;
             body.Position = 0;
 
-            var content = new StreamReader(body).ReadToEnd();
+            var content = new StreamReader(body, Encoding.Unicode).ReadToEnd();
```

Comment:
Assuming this doesn't cause problems without the BOM? Would be a little more reassuring if it was explicit in terms of endian-ness + no-bom